docs/zh/latest/plugin-develop.md

```diff
@@ -37,6 +38,53 @@ title: 插件开发
 - [编写测试用例](#编写测试用例)
   - [附上 test-nginx 执行流程](#附上-test-nginx-执行流程)
 
+此文档是关于 lua 语言的插件开发，其他语言请看：[external plugin](./external-plugin.md)。
+
+## 插件放置路径
+
+APISIX提供了两种方式来添加新的功能。
+
+1. 修改APISIX的源代码并重新发布 (不推荐)。
```

Comment:
```suggestion
1. 修改 Apache APISIX 的源代码并重新发布 (不推荐)。
```
There should be a space between Chinese and English, you could ref https://github.com/sparanoid/chinese-copywriting-guidelines .